mhs-reference-implementation/mhs/routing/tests/test_sds_handler.py

```diff
@@ -0,0 +1,63 @@
+from unittest import TestCase
+import mhs.routing.sds_handler as sds
+
+import mhs.routing.tests.ldap_mocks as mocks
+from utilities.test_utilities import async_test
+
+NHS_SERVICES_BASE = "ou=services, o=nhs"
+
+MHS_OBJECT_CLASS = "nhsMhs"
+
+OPENTEST_SDS_URL = "192.168.128.11"
+PARTY_KEY = " AP4RTY-K33Y"
+SERVICE_ID = "urn:nhs:names:services:psis:MCCI_IN010000UK13"
+ODS_CODE = "ODSCODE1"
+
+
+# A91461
+expected_mhs_attributes = {
+    'nhsEPInteractionType': 'HL7',
+    'nhsIDCode': 'ODSCODE1',
+    'nhsMHSAckRequested': 'always',
+    'nhsMHSActor': ['urn:oasis:names:tc:ebxml-msg:actor:toPartyMSH'],
+    'nhsMHSDuplicateElimination': 'always',
+    'nhsMHSEndPoint': ['https://vpn-client-1411.opentest.hscic.gov.uk/'],
+    'nhsMHSIsAuthenticated': 'transient',
+    'nhsMHSPartyKey': 'AP4RTY-K33Y',
+    'nhsMHSPersistDuration': 'PT5M',
+    'nhsMHSRetries': '2',
+    'nhsMHSRetryInterval': 'PT1M',
+    'nhsMHSSyncReplyMode': 'MSHSignalsOnly',
+    'nhsMHsIN': 'MCCI_IN010000UK13',
+    'nhsMHsSN': 'urn:nhs:names:services:psis',
+    # 'nhsMhsCPAId': 'S918999410559', # Exclude this value since there seems to be two responses causing errors on this
+    'nhsMhsFQDN': 'vpn-client-1411.opentest.hscic.gov.uk',
+    'nhsMhsSvcIA': 'urn:nhs:names:services:psis:MCCI_IN010000UK13',
+    'nhsProductKey': '7374',
+    # 'uniqueIdentifier': ['S918999410559']
+}
+
+
+class TestMHSAttributeLookupHandler(TestCase):
+    opentest_expected_reliability = {'nhsMHSAckRequested': 'always',
```

Comment:
Is this variable being used?

Reply:
Good point, it's no longer being used since we're using the full attribute set defined out of the class